packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,200 @@
+layout: Guides
+---
+
+<ApiLifecycle access="ea" />
+
+This guide explains how to build and submit an API service integration to the Okta Integration Network (OIN) for review and publishing.
+
+---
+
+**Learning outcomes**
+
+* Learn how to build an API service integration that can be published in the OIN.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+   > **Note:** Your org needs to have the API Service Integration feature enabled. Contact your Okta account team to enable the feature.
+* A service app that needs to access Okta APIs for your customer
+
+
+---
+
+## Overview
+
+A service-to-service app where where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
+API service integrations access Okta APIs using the OAuth 2.0 Client Credentials grant flow, where access is not associated with a user and resources can be restricted with scoped access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+
+> **Note:** Currently, Okta only supports OAuth APIs listed in [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints). We're working towards supporting scopes for all Okta API endpoints.
+
+If you have an API service integration for your app in the OIN, your customers can configure your integration to implement the OAuth 2.0 Client Credentials flow with their Okta tenant org. Each customer Okta org has its own authorization server. When a customer authorizes your API service integration to access their org, Okta generates a unique set of credentials (client ID and client secret) for that org.
+You must collect and store these credentials for each customer to allow your integration to work with your customer's Okta org.
+
+### API service integration Client Credentials grant flow
+
+At a high-level, the OAuth 2.0 Client Credentials flow for an API service integration has the following steps:
+
+1. Your customer's instance of your service app makes an authorization request to their Okta Authorization Server using their client credentials.
+
+   Your customer needs to register their instance of your app in their Okta org so that Okta can accept the authorization request. See [HOC&mdash;Set up an API service integration](https://www.figma.com/proto/1hqMxP4MrqXR07atucaGvM/OAuth-API-Integrations---Partner-Mocks?node-id=1550%3A9898&starting-point-node-id=1550%3A9668). After registration, the customer obtains their client credentials to [configure in your integration](#collect-and-save-customer-credentials). The customer's instance of your app can make an authorization request to Okta. See [Request for access token](#request-an-access-token).
```

Comment:
Maybe something like

> Your customer needs to authorize your integration in their Okta org. See [HOC&mdash;Set up an API service integration](https://www.figma.com/proto/1hqMxP4MrqXR07atucaGvM/OAuth-API-Integrations---Partner-Mocks?node-id=1550%3A9898&starting-point-node-id=1550%3A9668). After authorization, the customer obtains their client credentials to [configure in your integration](#collect-and-save-customer-credentials). See [Request for access token](#request-an-access-token).

? Trying to call these integrations rather than apps. We use the verbs "install and authorize" in the product. Might drop "install" but I think authorize is standard OAuth language?

Reply:
Updated section. Please review.